conda/plugins/subcommands/doctor/health_checks.py

```diff
@@ -30,6 +33,36 @@ def find_packages_with_missing_files(prefix: str | Path) -> dict[str, list[str]]
     return packages_with_missing_files
 
 
+def find_altered_packages(prefix: str | Path) -> list[str]:
+    """Finds altered packages"""
+    altered_packages = {}
+
+    def generate_sha256_checksum(filepath) -> str:
+        with open(filepath, "rb") as f:
+            bytes = f.read()
+            hash = hashlib.sha256(bytes).hexdigest()
+            return hash
+
+    prefix = Path(prefix)
+    for file in (prefix / "conda-meta").glob("*.json"):
+        data = json.loads(file.read_text())
+        required_data = data["paths_data"]["paths"]
+
+        for path in required_data:
+            _path = path.get("_path")
+            old_sha256 = path.get("sha256_in_prefix")
+            file_location = f"{prefix}/{_path}"
+            if os.path.isfile(file_location):
+                new_sha256 = generate_sha256_checksum(file_location)
+            else:
+                continue
```

Comment:
```suggestion
            if not os.path.isfile(file_location):
                continue
            new_sha256 = generate_sha256_checksum(file_location)
```

It's usually better to invert these statements to avoid an extra `else` clause if possible.